src/lib/staged_ledger/staged_ledger.ml

```diff
@@ -1225,7 +1225,7 @@ module T = struct
       { max_space= slots
       ; max_jobs= job_count
       ; commands_rev=
-          uc_seq
+          seq_rev uc_seq
```

Comment:
This would cause a double reverse on the transaction sequence in the second partition because the discarded commands from first diff computation are already in the reverse order. You could reverse `ts_seq` on line 1609 instead?

Reply:
Good catch, thanks! I've updated the code to store the discarded commands in their original order, so this rev will also do the right thing now.